container/jbossweb/pom.xml

```diff
@@ -71,7 +71,7 @@
         <profile>
             <id>AS7</id>
             <activation>
-                <activeByDefault>false</activeByDefault>
+                <activeByDefault>true</activeByDefault>
```

Comment:
I noticed the build is problematic while adding support for Tomcat 9. I have opened https://issues.jboss.org/browse/MODCLUSTER-485 to properly test all combinations. Also the installation instructions should be updated to build with and against proper jars.